shared/ui/src/commonMain/kotlin/com/adammcneilly/pocketleague/shared/ui/match/MatchDetailContent.kt

```diff
@@ -24,6 +25,12 @@ fun MatchDetailContent(
     games: List<GameDetailDisplayModel>,
     modifier: Modifier = Modifier,
 ) {
+    games.firstOrNull()?.let { game ->
+        if (!game.isPlaceholder) {
+            GameDetailDialog(game)
+        }
+    }
```

Comment:
Instead we should be handling based on a selected game property, and also clear that selected game on dismiss.